server/apps/a12n/views.py

```diff
@@ -0,0 +1,163 @@
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework_simplejwt.tokens import RefreshToken
+
+from .serializers import UserRegisterSerializer, UserLoginSerializer
+from .models import UserSession
+from apps.account.models import User
+
+from django.contrib.auth.tokens import default_token_generator
+from django.utils.http import urlsafe_base64_decode, urlsafe_base64_encode
+from django.utils.encoding import force_bytes, force_str
+from django.core.mail import BadHeaderError, send_mail
+
```

Comment:
Good practice to correctly sort your imports (read some resources).
U can use the `isort` tool that can refactor your import autormically or just check your code on errors.